src/main/java/org/example/RockPaperScissors/Processor.java

```diff
@@ -0,0 +1,57 @@
+package org.example.RockPaperScissors;
+
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import java.io.Serializable;
+
+
+
+
+@Slf4j
+public class Processor  {
```

Comment:
Z tego co zdazylem sie zorientowac to koncepcja processorow jest tym samym co DAO - maja agregowac operacje bazodanowe dla encji, nie robi sie nigdy ogolnego procesora tylko zawsze pod konkretna encje czyli powinniscie miec PlayerProcessor i RoundProcessor oddzielnie